src/clj_3df/compiler.cljc

```diff
@@ -32,6 +32,16 @@
 
 (def ^{:arglists '([pred] [pred coll])} separate (juxt filter remove))
 
+(defn remove-once [args coll]
```

Comment:
If this is intended to be a helper function, then please use `defn-` to keep it private to this namespace.

Reply:
Also this function would benefit greatly from a comment, I have no idea what it does :D